### PR TITLE
Add stochastic depth to conformer and share results on LibriSpeech 960h

### PR DESCRIPTION
**Pull request by pyf98**
Hi, in this PR, I added stochastic depth to `conformer_encoder` so that layer dropout can be used. I am also sharing two configs (with and without layer dropout) and their results for LibriSpeech 960h. As mentioned previously, the results are better than the current ones.

### CODE REVIEW

**Comment by pyf98** [comment]
Full results are coming soon. (Some decoding jobs are still running)

**Comment by csukuangfj** [comment]
How many parameters does the model have?

File: egs2/librispeech/asr1/README.md [hunk 1]
```diff
@@ -113,16 +113,166 @@
...
+- pytorch version: `pytorch 1.10.1`
+- Git hash: `c3569453a408fd4ff4173d9c1d2062c88d1fc060`
+  - Commit date: `Sun Mar 6 23:58:36 2022 -0500`
+
+## asr_train_asr_conformer9_layerdrop0.1_last6_raw_en_bpe5000_sp
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|beam60_ctc0.2/dev_clean|2703|54402|98.0|1.8|0.2|0.2|2.2|26.5|
```
Comment (sw005320) [inline egs2/librispeech/asr1/README.md:137]: How about only keeping the best setup?

Comment (pyf98) [reply egs2/librispeech/asr1/README.md:137]: ok, I can remove it

Comment (pyf98) [reply egs2/librispeech/asr1/README.md]: It's removed now

**Pull request merged**